lib/floki/raw_html.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Floki.RawHTML do
+  @moduledoc"""
```

Comment:
I prefer not to document modules that are "private" for now. I think the the docs from the "public" function are enough: https://hexdocs.pm/floki/Floki.html#raw_html/1
Could you please check it to `@moduledoc false`?